templates/base.html

```diff
@@ -35,9 +35,9 @@ <h3 class="masthead-title">
 
             <footer class="footer">
                 <hr>
-                <small>
+                <small>Join us on <a href="https://gitter.im/rust-osdev/Lobby" >Gitter.</a><br>
                     &copy; <time datetime="2020">2020</time>. All rights reserved.
-                    <a href="{{ get_url(path="@/pages/contact.md") | safe }}">Contact</a>
+                    <a href="{{ get_url(path="@/pages/contact.md") | safe }}">Contact</a> 
```

Comment:
Could you remove the trailing whitespace here?

Reply:
Ops, sorry about that!